4_cv_basics/3.5_Interpolation/nearest_neighbour/nearest_neighbour_functions.hpp

```diff
@@ -0,0 +1,255 @@
+/*
+Contains the Function definitions for Nearest-Neighbour Interpolation
+Functions for Scale Up/Down with specific number of reduction/increase of pixels or scaling by ratio are defined here
+They can be called from the main nearest_neighbour.cpp file
+*/
+#include "nearest_neighbour.hpp"
+
+using namespace std;
+using namespace cv;
+
+Mat nearest_neighbour_scale_image(Mat image, Mat scaled_image)
+{
+    /*
+    Nearest-Neighbour Interpolation main calculation loop
+    This function is called when image has to be reduced or enlarged with arguements
+        Mat Original Image
+        Mat New Image
+    And the function returns the resized image (Mat scaled_image)
+    */
```

Comment:
Add a line after multi line comment, similar for all functions implemented below.